connaisseur/validators/interface.py

```diff
@@ -8,10 +9,17 @@ def __init__(self, name: str, **kwargs):
         """
         pass
 
-    def validate(self, admission_request: AdmissionRequest, **kwargs) -> list:
+    def validate(self, image: Image, **kwargs) -> str:
         """
         Validates an admission request, using the extra arguments from the image policy.
 
         Returns a list of trusted digests.
         """
         raise NotImplementedError
+
+    @property
+    def healthy(self):
+        return True
```

Comment:
This should not be default return `True`, but validator implementation should have to do this explicitly. Let's `raise NotImplementedError` here.